app/controllers/merchant/items_controller.rb

```diff
@@ -1,6 +1,10 @@
 class Merchant::ItemsController < ApplicationController
   def index
     @merchant = Merchant.find(params[:merchant_id])
-    @items = Item.all 
+    @items = Item.all
+  end
+
+  def show
+    @item = Item.find(params[:id])
```

Comment:
If you ever need just a merchant's items on a view later, `@items = @merchant.items` could be useful to refactor in the future

Reply:
I don't think I understand... I only want one item here

Reply:
Oh, I changed to the full page view, I see what you mean now, I didn't make def index, I just accidentally deleted it and had to put it back